src/test/java/fact/io/CeresStreamTest.java

```diff
@@ -0,0 +1,62 @@
+package fact.io;
+
+import org.junit.Test;
+import stream.Data;
+import stream.io.SourceURL;
+
+import java.net.URL;
+
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+/**
+ * Created by Kai on 27.06.17.
+ */
+public class CeresStreamTest {
+
+    @Test
+    public void testFitsStream() {
+
+        try {
+            URL u =  FITSStreamTest.class.getResource("/ceres_output/18000/00018000.000_D_MonteCarlo018_Events.fits.gz");
```

Comment:
can we may be use the mc file from the resources or replace it with this one and an according runHeaders file. I think we shouldn't blow up the jar with too many fits files in the resources